sklearn/neighbors/_nth_element.pyx

```diff
@@ -0,0 +1,62 @@
+# distutils : language = c++
+
```

Comment:
I think we need a comment here to describe that this file is creating an Comparator so that we can use `nth_element`.